### PR TITLE
Fix load-balancing demo application paths

### PR DESCRIPTION
**Pull request by jajik**
The demo application seems working with a few tweaks.

ref #141

### CODE REVIEW

File: docs/src/main/asciidoc/demo.adoc [hunk 1]
```diff
@@ -3,8 +3,6 @@
 
 TIP: {editurl}demo.adoc[{editnote}]
 
-WARNING: The demo application has not been recently updated and might not work with newer software versions as described below!
```
Comment (rhusar) [inline docs/src/main/asciidoc/demo.adoc:6]: Actually this is what I added last week :-D

Are you saying it works no problem on WildFly? I doubt it!

**Review by rhusar — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/demo.adoc (hunk 1 above)
Comment (jajik) [reply docs/src/main/asciidoc/demo.adoc:6]: Hm, sorry, I was a bit fast with the removal. I'll add it back.

Tomcat (8.5, 9) works.

Thanks for the catch.

Comment (jajik) [reply docs/src/main/asciidoc/demo.adoc]: (Thinking about it, I honestly don't know why I separated this from #144...)

Comment (rhusar) [reply docs/src/main/asciidoc/demo.adoc]: Right, old tomcats should work ,but anything jakarta, nope..

**Review by rhusar — APPROVED** [review]

**Comment by rhusar** [comment]
The demo should be moved out of the main repo anyway, so we will update this even more then.

**Pull request merged**